### ISSUE TITLE
"fly configure" usage is missing correct syntax in fly help

### ISSUE THREAD

**Issue by jbayer** [concourse/concourse#64]
the description shows how to name a pipeline, but the usage shows "command" instead of "fly" and is missing the PIPELINE_NAME.

i would expect to see:
```
USAGE:
   fly configure PIPELINE_NAME [command options] [arguments...]
```

but i actually see:

```
$ fly h c
NAME:
   configure - Update configuration

USAGE:
   command configure [command options] [arguments...]

DESCRIPTION:
   Specify a pipeline name to configure via `fly configure your-pipeline-name-here`

OPTIONS:
   --config, -c 						pipeline configuration file
   --json, -j							print config as json instead of yaml
   --var, -v [--var option --var option]			variable flag that can be used for filling in template values in configuration (i.e. -var secret=key)
   --vars-from, --vf [--vars-from option --vars-from option]	variable flag that can be used for filling in template values in configuration from a YAML file
   --paused							should the pipeline start out as paused or unpaused (true/false)

```

**Comment by krishicks** [comment]
This is the wrong place for this issue. It could be on [fly](https://github.com/concourse/fly), but it could also be on [codegangsta/cli](https://github.com/codegangsta/cli/blob/f9982619ccace7df963bb7f6d1de3a6f014ff709/help.go#L44), as that's what prints the help text.

**Comment by xoebus** [comment]
Yep, the library is good at some things but generating help text is not one of them. We'll probably replace it at some point.

**Comment by jchesterpivotal** [comment]
Looks like I [logged a dupe](https://github.com/concourse/fly/issues/40) on the fly repo.

**Comment by vito** [comment]
Forgot to follow up on this; should be fixed in 0.66.0+. The new command is `set-pipeline`.
